lib/mongoid/history/services/options_cleaner.rb

```diff
@@ -0,0 +1,73 @@
+module Mongoid
+  module History
+    class OptionsCleaner
```

Comment:
I think a better name could be just `Options`, no? We're not really cleaning anything, just dealing with options.